src/main/java/appeng/container/implementations/ContainerCraftConfirm.java

```diff
@@ -67,10 +69,10 @@
 public class ContainerCraftConfirm extends AEBaseContainer
 {
 
-	public final ArrayList<CraftingCPURecord> cpus = new ArrayList<CraftingCPURecord>();
-	final ITerminalHost priHost;
-	public Future<ICraftingJob> job;
-	public ICraftingJob result;
+	private final ArrayList<CraftingCPURecord> cpus = new ArrayList<CraftingCPURecord>();
+	private final ITerminalHost priHost;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "priHost" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)